docs/internships/gsoc/application-process.md

```diff
@@ -3,33 +3,33 @@ id: gsoc-application-process
 title: Applying
 ---
 
-Details are announced now, and we are in the project discussion phase. We encourage you to discuss you application with the project community.
+Details are announced now, and we are in the project discussion phase. We encourage you to discuss your application with the project community.
 
 ## Introduction
 
-In order to apply we ask that you create:
+To apply we ask that you create:
 
 1. a list of deliverables, quantifiable results for the Palisadoes community,
 1. a detailed description / design document,
 1. an approach,
 1. an approximate schedule and
 1. something of a background text.
-1. We also need you to let us know what other commitments you have for the period of GSoC, for example, do you have any exams or a part time job? In other words &#8211; sell yourselves, as if this was an actual job.
+1. We also need you to let us know what other commitments you have for the period of GSoC, for example, do you have any exams or a part-time job? In other words &#8211; sell yourselves, as if this was an actual job.
 
-In drawing up your proposal you should work with your chosen projects community.
+In drawing up your proposal you should work with your chosen project community.
```

Comment:
In drawing up your proposal you should work with your chosen project's community.